plugins/org.yakindu.sct.generator.c/library/FeatureTypeLibrary.xmi

```diff
@@ -11,13 +11,6 @@
     <parameters
         name="moduleName"
         optional="true"/>
-    <parameters
-        name="statemachinePrefix"
-        optional="true"/>
-    <parameters
-        name="maxIdentifierLength"
```

Comment:
Keep this if we can add  validation rule that always attaches a warning:

'This property is not supported any longer and will be ignored. Please use @shortcidentifier annotation in C statecharts'